app/assets/javascripts/discourse/ember-cli-build.js

```diff
@@ -105,7 +105,7 @@ module.exports = function (defaults) {
 
   // WARNING: We should only import scripts here if they are not in NPM.
   // For example: our very specific version of bootstrap-modal.
-  app.import(vendorJs + "bootbox.js");
+  app.import("node_modules/bootbox/bootbox.min.js");
```

Comment:
Is there any way we can rely on ember-auto-import to take care of this, rather than manually importing from `node_modules`? 🤔 

(especially given the WARNING comment in the previous line ;) )

Reply:
I'm going to scrap this PR for now. TIL: we effectively forked bootbox at 3.2.0 - see the history of the file: https://github.com/discourse/discourse/commits/main/vendor/assets/javascripts/bootbox.js

…so we can't replace it with the npm version or upgrade it easily. There is a topic on dev about bootbox and the modal mixin. For now we'll have to live with a vendor file.